src/dictionaries.cpp

```diff
@@ -49,6 +49,14 @@ Dictionaries::Dictionaries(const std::string& filePath)
     std::string line;
     size_t lineNumber = 0;
     bool vendorFlag = false;
```

Comment:
Do you still need `vendorFlag` if you have `vendorName`? I mean, you can replace `vendorFlag` check by seeing if the `vendorName` is empty or not.